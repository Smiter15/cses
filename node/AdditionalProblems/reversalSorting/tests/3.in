10
6 4 8 2 10 1 9 3 5 7
