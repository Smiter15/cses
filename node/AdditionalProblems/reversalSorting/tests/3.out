9
1 6
2 3
3 8
4 6
5 9
6 9
7 10
8 9
9 10
